app/src/main/java/org/helfoome/presentation/MainActivity.kt

```diff
@@ -119,6 +119,10 @@ class MainActivity : BindingActivity<ActivityMainBinding>(R.layout.activity_main
             tvNumber.setOnClickListener {
                 startActivity(Intent(Intent.ACTION_DIAL, Uri.parse("tel:" + tvNumber.text)))
             }
+
+            binding.btnHamburger.setOnClickListener {
+                binding.layoutDrawer.openDrawer(binding.layoutDrawerHeader.nvHeader)
+            }
```

Comment:
스코프 함수 이용하면 좀 더 깔끔하게 표현 가능할 것 같아요!

**ex.**
```suggestion
            with(binding) {
            btnHamburger.setOnClickListener {
                layoutDrawer.openDrawer(layoutDrawerHeader.nvHeader)
            }
            }
```

Reply:
그리고 drawer에 연동된(그 가려지는 부분이요!) 레이아웃이 하나라면 그냥 open()만 호출해줘도 괜찮아요!

이런식으로요!

`binding.layoutDrawer.open()`

Reply:
그리고 네비게이션뷰의 경우 layout 헤더의 경우 xml에서 `app:headerLayout` 속성을 이용해사 헤더를 집어넣는 것도 가능한데 왜 코드단에서 이런식으로 구현하셨는지 알 수 있을까요?

Reply:
좋습니당~~